.travis.yml

```diff
@@ -1,16 +1,35 @@
 language: go
-go:
-  - 1.x
-
-sudo: required
-services:
-  - docker
-
+sudo: false
 env:
-  - GOVERSION=1.11
-  - GOVERSION=1.12
-
-install: true
-
+  - GO111MODULE=on
+matrix:
+  include:
+    - os: linux
+      go: 1.12.x
+      cache:
+        directories:
+          - $HOME/.cache/go-build
+          - $HOME/go/pkg/mod
+    - os: linux
+      go: 1.11.x
+      cache:
+        directories:
+          - $HOME/.cache/go-build
+          - $HOME/go/pkg/mod
+    - os: osx
+      go: 1.12.x
+      cache:
+        directories:
+          - $HOME/.cache/go-build
+          - $HOME/go/pkg/mod
+    - os: osx
+      go: 1.11.x
+      cache:
+        directories:
+          - $HOME/Library/Caches/go-build
+          - $HOME/go/pkg/mod
+install:
+  - go get -v github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
is this used for anything? i don't see it anywhere else in the scripts or the travis output except for this installation step.

Reply:
nope, i dropped it.